core/unix/coredump.c

```diff
@@ -274,6 +273,23 @@ mcontext_to_user_regs(DR_PARAM_IN priv_mcontext_t *mcontext,
 #endif
 }
 
+/*
+ * Return the entry point of the program.
+ */
+uint64_t
+get_entry_point(DR_PARAM_IN priv_mcontext_t *mcontext)
+{
+#ifdef DR_HOST_NOT_TARGET
+    return 0;
+#elif defined(X86)
+    return (uint64_t)mcontext->rip;
+#elif defined(AARCH64)
+    return (uint64_t)mcontext->pc;
```

Comment:
Isn't `pc` a unioned name that works for all arches, so we don't need any ifdefs here at all?

Reply:
Done. I've removed the function.